modules/ROOT/pages/endpoints.adoc

```diff
@@ -12,7 +12,11 @@ Partner Manager uses endpoints to:
 
 == Endpoint Ownership
 
-You can assign the host or a partner as the owner of partner-facing endpoints to control the use of endpoints within inbound and outbound message flows. When creating a message flow, endpoints owned by:
+You can assign the host or a trading partner as the owner of partner-facing endpoints to control the use of endpoints within inbound and outbound message flows.
+
+When creating partner-facing endpoints directly from the host or a trading partner, the ownership of the endpoint is automatically assigned to the host or trading partner from which the endpoint was created.
+
+When creating partner-facing endpoints from the Message Flow wizard, you can assign the endpoint to either the host or the trading partner. In a message flow, endpoints owned by:
```

Comment:
It's a bit hard to read the bullets because the syntax of a single sentence is broken across the intro & the bullets. (I know this existed before your update). Also because we change from talking about assignments to owning (I assume they mean the same thing).

Consider simplifying the syntax either now or in a later edit similar to the following for lines 19-22:

In a message flow, availability depends on the endpoint assignment:

* Any trading partner can select an endpoint assigned to the host.
* Only the same trading partner can select an endpoint assigned to that trading partner

Reply:
That is much better. Thank you! Changes implemented.